kubebb/core
# refactor: update pipelines for rating

#### What type of PR is this?
/kind refactor


#### What this PR does / why we need it

#### Which issue(s) this PR fixes

Fixes #

#### Special notes for your reviewer


To test this rating pipelines ,pls follow these steps:

1. make sure kubebb/core and tekton installed

2. apply rbac
```shell
kubectl apply -f pipeline/rating/rbac
```

3. apply tasks
```shell
kubectl apply -f pipeline/rating/security
kubectl apply -f pipeline/rating/reliability
```

4. test with samples

```shell
kubectl apply -f pipeline/rating/samples
```

5. check results

```shell
kubectl get pods -nkubebb-system
kubectl get pipelineruns -nkubebb-system
```

6. Test with different namespace

1) sync service account and rolebinding
```shell
kubectl kustomize pipeline/rating/rbac | kubectl apply -f -
```

2) apply rating runs
```shell
kubectl kustomize samples |  kubectl apply -f -
```
